[2022-03-17 19:47:31,671] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_project.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-03-17 19:47:31,681] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_project.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-03-17 19:47:31,681] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 19:47:31,681] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-03-17 19:47:31,682] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 19:47:31,692] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-01 00:00:00+00:00
[2022-03-17 19:47:31,698] {standard_task_runner.py:52} INFO - Started process 610 to run task
[2022-03-17 19:47:31,702] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_project', 'local_to_gcs_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_project.py', '--cfg-path', '/tmp/tmp0bv8otbj', '--error-file', '/tmp/tmpt29aeo33']
[2022-03-17 19:47:31,703] {standard_task_runner.py:77} INFO - Job 22: Subtask local_to_gcs_task
[2022-03-17 19:47:31,765] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_project.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host 06aea6ade097
[2022-03-17 19:47:31,810] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-03-17 19:47:31,833] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_project
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-03-17 19:47:32,838] {python.py:175} INFO - Done. Returned value was: None
[2022-03-17 19:47:32,850] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_project, task_id=local_to_gcs_task, execution_date=20190101T000000, start_date=20220317T194731, end_date=20220317T194732
[2022-03-17 19:47:32,878] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-17 19:47:32,913] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-03-20 06:18:12,432] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_project.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-03-20 06:18:12,444] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_project.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-03-20 06:18:12,444] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 06:18:12,444] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-03-20 06:18:12,444] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 06:18:12,456] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-01 00:00:00+00:00
[2022-03-20 06:18:12,461] {standard_task_runner.py:52} INFO - Started process 374 to run task
[2022-03-20 06:18:12,466] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_project', 'local_to_gcs_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_project.py', '--cfg-path', '/tmp/tmpabwxg3yz', '--error-file', '/tmp/tmpga4vaxtd']
[2022-03-20 06:18:12,467] {standard_task_runner.py:77} INFO - Job 42: Subtask local_to_gcs_task
[2022-03-20 06:18:12,533] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_project.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host 974079c1cd0d
[2022-03-20 06:18:12,576] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-03-20 06:18:12,599] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_project
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-03-20 06:18:13,759] {python.py:175} INFO - Done. Returned value was: None
[2022-03-20 06:18:13,771] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_project, task_id=local_to_gcs_task, execution_date=20190101T000000, start_date=20220320T061812, end_date=20220320T061813
[2022-03-20 06:18:13,803] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-20 06:18:13,837] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
